client/src/modules/detailedView/Contact/Contact.tsx

```diff
@@ -4,8 +4,16 @@ import Navbar from '../../layouts/Navbar/Navbar';
 import DetailedViewHeader from '../DetailedViewHeader/DetailedViewHeader';
 import ContactContent from './ContactContent/ContactContent';
 import ContactSidebar from './ContactSidebar/ContactSidebar';
+import { loadContactById } from '../../layouts/Contacts/People/contactActions';
+import { RouteComponentProps } from 'react-router';
+import ContactModel from '../../../models/Contact';
 
-class Contact extends React.Component {
+interface Props extends RouteComponentProps<{contactId: string}>{
+  contact: ContactModel;
+
+  loadContactById(contact: string): void;
```

Comment:
in the name of the param should be mentioned that it is an id, but not a contact